<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>avoc</class>
 <widget class="QMainWindow" name="avoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1.3px solid rgb(200, 141, 40);
border-radius: 40px;
background-color: rgb(23, 46, 78);
border-radius: 9px;
padding: 0 2px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>1121</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-top: 5px solid white ;
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar not QTabWidget */
QTabBar::tab {
background: #FFFFFF;
border: 2px solid #172e4e;
border-bottom-color: #172e4e; /* same as the pane color */
border-top-left-radius:8px;
border-top-right-radius: 8px;
min-width: 140px;
padding: 8px;
color: 
#172e4e;


}
QTabBar::tab:selected, QTabBar::tab:hover {
background:
rgb(51, 105, 175)}
QTabBar::tab:selected {
border-color: #87CEEB;
border-bottom-color: #87CEEB; /* same as pane color */
color: white;
;

}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}
QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CRUD</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>591</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLineEdit" name="le_cin">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>31</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_num">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>120</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_adresse">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>180</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>19</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Grade</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>119</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Numéro de téléphone</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>170</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
      <widget class="QComboBox" name="le_grade">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>71</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>Penaliste</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Avocat d'affaires</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Avocat d'affaires civiles</string>
        </property>
       </item>
      </widget>
      <widget class="QDateEdit" name="la_date">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>21</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>227</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>227</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tab_avocat">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>461</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Saisir les données de l'avocat à ajouter/Saisir les modifications en conservant le CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>350</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Saisir le CIN de l'avocat à supprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Suppression</string>
      </property>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>77</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin_supp">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: rgb(250, 235, 215);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>le CIN</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pb_PDF">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_tri1">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tri par nom</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_tri2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>69</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tri par prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="la_recherche">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_rech">
      <property name="geometry">
       <rect>
        <x>852</x>
        <y>10</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_stat">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
      </property>
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>430</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
      </property>
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>NotePad/Camera</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>NOTEPAD</string>
      </property>
      <widget class="QPushButton" name="note">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
       </property>
       <property name="text">
        <string>NotePad</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cliquer ici pour acceder a votre notepad</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Camera</string>
      </property>
      <widget class="QPushButton" name="cam">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

color: #000000;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cliquer ici pour acceder a la camera</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: #778899c;
background: #87CEEB;
border: none;
border: 1px solid  #172e4e;
border-radius: 8px;

}
QPushButton:hover {
color:#ffffff;

background:#778899;
border: none;
}</string>
      </property>
      <property name="text">
       <string>Fermer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
